ACPs/103-dynamic-fees/README.md

```diff
@@ -0,0 +1,135 @@
+```text
+ACP: 103
+Title: Add Dynamic Fees to the X-Chain and P-Chain
+Author(s): Dhruba Basu (https://github.com/dhrubabasu), Alberto Benegiamo (https://github.com/abi87), Stephen Buttolph (https://github.com/StephenButtolph)
+Discussions-To: <GitHub Discussion URL (POPULATED BY MAINTAINER, DO NOT SET)>
+Status: Proposed
+Track: Standards
+```
+
+## Abstract
+
+Introduce a dynamic fee mechanism to the X-Chain and the P-Chain. Preview a future transition to a multidimensional fee mechanism.
+
+## Motivation
+
+Under a fixed fee mechanism, there is a greater likelihood of higher network congestion. During periods of high demand, there is no economic incentive for users to withhold issuance of their transactions, extending the period of high demand.
+
+The C-Chain, in [Apricot Phase 3](https://medium.com/avalancheavax/apricot-phase-three-c-chain-dynamic-fees-432d32d67b60), employs a dynamic fee mechanism to raise the price during periods of high demand and lowering the price during periods of low demand. As the price gets too expensive, network utilization will decrease, which drops the price.
+
+The X-Chain and P-Chain currently operate under a fixed fee mechanism. To better handle increased load, they should be migrated to a dynamic fee mechanism. This migration should be done in tandem with [ACP-77](../77-reinventing-subnets/README.md) to ensure developers can account for the dynamic fee mechanism.
```

Comment:
Why are we brining up ACP 77 here? This ACP is unrelated to 77 afaict (77 should bring up this ACP as it will use this mechanism for it's fee)

Reply:
good point- removed